funnel/views/session.py

```diff
@@ -93,15 +93,15 @@ def view(self):
         return dict(project=self.obj.project, active_session=session_data(self.obj, with_modal_url='view_popup'),
             from_date=date_js(self.obj.project.date), to_date=date_js(self.obj.project.date_upto),
             sessions=session_list_data(self.obj.project.scheduled_sessions, with_modal_url='view_popup'),
-            timezone=timezone(self.obj.project.timezone).utcoffset(datetime.now()).total_seconds(),
+            timezone=self.obj.project.timezone.utcoffset(datetime.utcnow()).total_seconds(),
             venues=[dict(venue.current_access()) for venue in self.obj.project.venues],
             rooms=dict([(room.scoped_name, {'title': room.title, 'bgcolor': room.bgcolor}) for room in self.obj.project.rooms]))
 
     @route('viewsession-popup')
     @render_with('session_view_popup.html.jinja2')
     @requires_permission('view')
     def view_popup(self):
-        return dict(session=self.obj, timezone=self.obj.project.timezone, localize_date=localize_date)
+        return dict(session=self.obj, timezone=self.obj.project.timezone.zone, localize_date=localize_date)
```

Comment:
The `localize_date` function shouldn't be passed around like this. It should be registered as a Jinja2 template helper.